python-lib/plugin_io_utils.py

```diff
@@ -97,6 +106,16 @@ def move_columns_after(
     return df.reindex(columns=reordered_columns)
 
 
+def get_keyword(text: AnyStr, case_insensitive: bool) -> AnyStr:
+    """Return text in its wanted-case form"""
+    return text.lower() if case_insensitive else text
+
+
+def get_sentence(span: Span, case_insensitive: bool) -> Union[Span, Doc]:
```

Comment:
This would be better placed as a static method in a NLP module

Reply:
As I said here https://github.com/dataiku/dss-plugin-nlp-analysis/pull/14#discussion_r620472517 I put it with get_keyword method in a nlp_utils.py file